src/fastify.ts

```diff
@@ -57,7 +49,7 @@ const start = async () => {
   await registerAppPlugins(instance);
 
   try {
-    await instance.listen({ port: PORT, host: HOSTNAME });
+    await instance.listen({ port: PORT, host: '0.0.0.0' });
```

Comment:
Are we sure we want to hardcode this?

Reply:
I can add another variable that controls the host address that the server listens to. But I think in general this should be fine to use in most situations. Instead I will set it as the default.